packages/shared/lib/auxiliary/icon/constants/settings-icon-svg.ts

```diff
@@ -1,28 +1,29 @@
-import { AdvancedSettings, GeneralSettings, HelpAndInfo, SecuritySettings } from '@lib/core/router'
+import { AdvancedSettingsRoute, GeneralSettingsRoute, HelpAndInfoRoute, SecuritySettingsRoute } from '@core/router'
+
 import { Icon } from '../enums'
 
 export const SETTINGS_ICON_SVG = {
-    [GeneralSettings.Theme]: Icon.Theme,
-    [GeneralSettings.Language]: Icon.Language,
-    [GeneralSettings.Currency]: Icon.Currency,
-    [GeneralSettings.Notifications]: Icon.Bell,
-    [GeneralSettings.NetworkStatus]: Icon.View,
-    [GeneralSettings.ChangeProfileName]: Icon.Profile,
-    [SecuritySettings.ExportStronghold]: Icon.Export,
-    [SecuritySettings.AppLock]: Icon.Lock3,
-    [SecuritySettings.ChangePassword]: Icon.Lock2,
-    [SecuritySettings.ChangePincode]: Icon.Lock,
-    [SecuritySettings.DeleteProfile]: Icon.Delete,
-    [AdvancedSettings.NetworkConfiguration]: Icon.Wifi,
-    [AdvancedSettings.DeepLinks]: Icon.Link,
-    [AdvancedSettings.WalletFinder]: Icon.Reset,
-    [AdvancedSettings.HiddenAccounts]: Icon.View,
-    [AdvancedSettings.ErrorLog]: Icon.Warning,
-    [AdvancedSettings.CrashReporting]: Icon.Doc,
-    [AdvancedSettings.Diagnostics]: Icon.Tools,
-    [AdvancedSettings.MigrateLedgerIndex]: Icon.Ledger,
-    [HelpAndInfo.Documentation]: Icon.Doc,
-    [HelpAndInfo.Faq]: Icon.Speech,
-    [HelpAndInfo.Discord]: Icon.Discord,
-    [HelpAndInfo.ReportAnIssue]: Icon.Help,
+    [GeneralSettingsRoute.Theme]: Icon.Theme,
+    [GeneralSettingsRoute.Language]: Icon.Language,
+    [GeneralSettingsRoute.Currency]: Icon.Currency,
+    [GeneralSettingsRoute.Notifications]: Icon.Bell,
+    [GeneralSettingsRoute.NetworkStatus]: Icon.View,
+    [GeneralSettingsRoute.ChangeProfileName]: Icon.Profile,
+    [SecuritySettingsRoute.ExportStronghold]: Icon.Export,
+    [SecuritySettingsRoute.AppLock]: Icon.Lock3,
+    [SecuritySettingsRoute.ChangePassword]: Icon.Lock2,
+    [SecuritySettingsRoute.ChangePincode]: Icon.Lock,
+    [SecuritySettingsRoute.DeleteProfile]: Icon.Delete,
+    [AdvancedSettingsRoute.NetworkConfiguration]: Icon.Wifi,
+    [AdvancedSettingsRoute.DeepLinks]: Icon.Link,
+    [AdvancedSettingsRoute.WalletFinder]: Icon.Reset,
+    [AdvancedSettingsRoute.HiddenAccounts]: Icon.View,
+    [AdvancedSettingsRoute.ErrorLog]: Icon.Warning,
+    [AdvancedSettingsRoute.CrashReporting]: Icon.Doc,
+    [AdvancedSettingsRoute.Diagnostics]: Icon.Tools,
+    [AdvancedSettingsRoute.MigrateLedgerIndex]: Icon.Ledger,
+    [HelpAndInfoRoute.Documentation]: Icon.Doc,
+    [HelpAndInfoRoute.Faq]: Icon.Speech,
+    [HelpAndInfoRoute.Discord]: Icon.Discord,
+    [HelpAndInfoRoute.ReportAnIssue]: Icon.Help,
```

Comment:
They were renamed with `Route` to make all of the code within the `router` module consistent. They aren't technically routes, so the `Route` keyword doesn't make sense, although it would be nice to have that consistency. I noticed we use them as "child routes" in the settings' router components ([example](https://github.com/iotaledger/firefly/blob/stardust-develop/packages/shared/routes/dashboard/settings/views/advanced/Advanced.svelte#L23)) - maybe we could use `ChildRoute` instead?

Reply:
Also, the settings views could probably be refactored similarly to onboarding in this PR - establish `ChildRouter` components for the settings.

Reply:
My only thing is that even then, these still aren't technically "routes" even if designated as a "child route".